diskover.py

```diff
@@ -684,7 +684,8 @@ def index_create(indexname):
 
     # check plugins for additional mappings
     for plugin in plugins:
-        mappings = (plugin.add_mappings(mappings))
+        if getattr(plugin, '__version__', 1) == 1:
```

Comment:
This concerns me a bit, because the dynamic fields won't be included in the index mapping so ElasticSearch will just guess their 'type'.  It's generally pretty good at guessing, but it would be nice if we could be more efficient.